src/css/components/_c-form.scss

```diff
@@ -101,6 +101,27 @@ $_c-input-margin-top: 1.5rem;
 }
 
 
+.c-form__error-wrapper {
+	border: 4px solid $color-red;
```

Comment:
Could we use a variable from [`_variables.borders.scss`](https://github.com/a11yproject/a11yproject.com/blob/main/src/css/logic/_variables.borders.scss) here? `$border-thin` seems like it might work.